README.md

```diff
@@ -1,39 +1,90 @@
 # Numeric Separators
 
-This is a [strawman proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
+## Stage 1
+
+This is a [proposal](https://tc39.github.io/process-document/) to introduce a separator in numeric literals in [ECMAScript](https://github.com/tc39/ecma262/).
 
 ## Motivation
 
-This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits.
+This feature enables developers to make their numeric literals more readable by creating a visual separation between groups of digits. 
+
+
+## Examples
+
+(The following examples also appear in the README.md of Babel transform plugin for this proposal.)
+
+
+### Decimal Literals
+
+```js
+let budget = 1_000_000_000_000;
+
+// What is the value of `budget`? It's 1 trillion!
+// 
+// Let's confirm:
+console.log(budget === 10 ** 12); // true
+```
+
+### Binary Literals
+
+```js
+let nibbles = 0b1010_0001_1000_0101;
+
+// Is bit 7 on? It sure is!
+// 0b1010_0001_1000_0101
+//             ^
+//             
+// We can double check: 
+console.log(!!(nibbles & (1 << 7))); // true
+```
 
-For example:
+### Hex Literal
 
 ```js
-var thousands = 10_000; // Instead of 10000.
-var credit_card_number = 1234_5678_9012_3456; // Instead of 123456789012345.
-var social_security_number = 999_99_9999; // Instead of 999999999.
-var pi = 3.14_15; // Instead of 3.1415
-var bytes = 0b11010010_01101001_10010100_10010010; // Instead of 0b11010010011010011001010010010010.
-var 0xCAFE_F00D; // Instead of 0XCAFEF00D.
+// Messages are sent as 24 bit values, but should be 
+// treated as 3 distinct bytes:
+let message = 0xA0_B0_C0;
+
+// What's the value of the upper most byte? It's A0, or 160.
+// We can confirm that:
+let a = (message >> 16) & 0xFF; 
```

Comment:
They are very much related and they are long on purpose: they illustrate the visibility/readability  advantage that numeric separator offers. The readme effectively _is_ the introductory doc—we still need to write the actual spec doc.